src/SprykerSdk/Acp/Console/AbstractConsole.php

```diff
@@ -99,12 +99,25 @@ protected function getFacade(): AcpFacadeInterface
      *
      * @return void
      */
-    protected function printMessages(OutputInterface $output, ArrayObject $messageTransfers): void
+    protected function printMessagesVerbose(OutputInterface $output, ArrayObject $messageTransfers): void
```

Comment:
Why do we have this change? We should print messages only by using `-v` with the console execution.

Reply:
I asked you in slack, it is solution for it
<img width="533" alt="Screenshot 2022-05-30 at 13 35 46" src="https://user-images.githubusercontent.com/86324025/170974805-429506fc-fc52-4c61-94e5-2c5503882596.png">